### PR TITLE
OpenMedia.4KStogram version 3.4.3.3630

### PR DESCRIPTION
**Pull request by KaranKad**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22085)

### CODE REVIEW

**Comment by ImJoakim** [comment]
image

This also occurs with all currently listed versions.

```
PS C:\Users\WDAGUtilityAccount\Desktop\manifests> winget install 4kstogram
Found 4K Stogram [OpenMedia.4KStogram]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://dl.4kdownload.com/app/4kstogram_3.4.2_x64.msi
An unexpected error occurred while executing the command:
InternetOpenUrl() failed.
0x80072f05 : unknown error
```

**Comment by KaranKad** [comment]
Looks like their certificate expired or something.

**Comment by ImJoakim** [comment]
> Looks like their certificate expired or something.

Indeed 

image

**Comment by ImJoakim** [comment]
@KaranKad I sent them an email yesterday, it should be fixed now. Go ahead and close and reopen the PR to re-run it.

**Comment by denelon** [comment]
@ImJoakim and @KaranKad. Just a heads-up. I'll be off the grid next week **7/26-7/30**: Training at 🏕️ Baker.

**Comment by ImJoakim** [comment]
> @ImJoakim and @KaranKad. Just a heads-up. I'll be off the grid next week **7/26-7/30**: Training at 🏕️ Baker.

That sounds fun, enjoy yourself! It's important to take some time off 👍

File: manifests/o/OpenMedia/4KStogram/3.4.3.3630/OpenMedia.4KStogram.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: OpenMedia.4KStogram
+PackageVersion: 3.4.3.3630
+PackageLocale: en-US
+Publisher: OpenMedia LLC
```
Comment (OfficialEsco) [inline manifests/o/OpenMedia/4KStogram/3.4.3.3630/OpenMedia.4KStogram.locale.en-US.yaml:7]:
```suggestion
Publisher: Open Media LLC
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
image

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**